internal/buildinfo/version.go

```diff
@@ -8,17 +8,126 @@ import (
 	"time"
 
 	"github.com/blang/semver"
+	"github.com/loadsmart/calver-go/calver"
 	"github.com/superfly/flyctl/terminal"
 )
 
+type Version interface {
+	String() string
+	EQ(other Version) bool
+	Outdated(string) (bool, error)
```

Comment:
It is a bit odd to see that Outdated takes two versions. I was expecting something like below.

```
v, _ := buildinfo.ParseSemver("0.0.1")
outdated, _ := v.Outdated() // returns true
```

How about calling this method LT (like semver) or Before (like time.Time)?

Also how about taking Version instead of string, to be more like EQ()?

Reply:
IMO it makes sense to operate on two versions, but I agree that it could be called LT or Before just to match convention